engine/trainer.py

```diff
@@ -738,6 +758,12 @@ def __init__(self, **kwargs):
             stats = wrapper_matching_dataset_lazy(all_matching_stats, self.cfg.TEST.MATCHING_STATS_THS)
             if self.cfg.TEST.VORONOI_ON_MASK:
                 stats_vor = wrapper_matching_dataset_lazy(all_matching_stats_voronoi, self.cfg.TEST.MATCHING_STATS_THS)
+                
+                
+        if self.cfg.TEST.MATCHING_VJI_PAI and self.cfg.PROBLEM.TYPE == 'INSTANCE_SEG' and self.cfg.DATA.TEST.LOAD_GT:
+            stats = wrapper_matching_VJI_and_PAI(all_matching_stats)
```

Comment:
Same problem with variable names: change them so that  do not match with the ones used in matching stats.